Cesium.Compiler/stdlib/limits.h

```diff
@@ -3,6 +3,14 @@
 #define SCHAR_MAX     127
 #define UCHAR_MAX     0xff
 
+#ifndef CESIUM_CHAR_UNSIGNED
```

Comment:
To be honest, I can't see why we need such an option. Why not just stick with the standard (which allows us to do an arbitrary choice and keep it)?

Supporting this will be a headache. I'd prefer us to stick with the unsigned variant for now, and ditch the signed one.

Too many compiler options (which we have to support, no less!) would be bad at this point of Cesium lifetime.

Reply:
By default char is signed on major platforms. I suspect that would be headache to compile code with deviation from what most compilers expect.

I know that this is speculative change, because I add it just to test `#if/#else/#endif`. So can remove.

Reply:
One of the minor purposes of Cesium is to invent a "better" (in some terms, ha-ha) C implementation: say, we have `long` and `int` different which is not in line with some other implementations.

I think that making `char` unsigned (and thus more in line with `System.Byte`) is one of these points. Interestingly, we even have this documented: `char` maps to `System.Byte` and not `System.SByte`.

In the future, we may try changing the decision if it shows itself problematic, but for now I'd keep it as-is. `char` should be unsigned for now.

Reply:
One other absolutely heretical thought I'd like us to go along with is [having `long long` an actual 128-bit type](https://github.com/ForNeVeR/Cesium/issues/127) instead of what everyone else is doing :)